src/rpc_server.rs

```diff
@@ -229,6 +226,35 @@ fn get_accounts(
     Ok(vec![])
 }
 
+/*
+ * The method takes an ownership of requests instance and extracts all valuable items out
+ * returns back:
+ * transaction: Option<String>
+ * accounts: Option<Vec<String>>
+ * options: GetPriorityFeeEstimateOption
+ */
+fn extract_data(
```

Comment:
this method takes ownership of request and extracts the transaction / accounts and options out (creates a default options if options were not provided). I found it easier to work with extracted properties than try to navigate the request